Packs/CaseManagement-Generic/Scripts/LinkIncidentsButton/LinkIncidentsButton.py

```diff
@@ -0,0 +1,15 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+linkedIncidentIDs = demisto.args().get("linkedIncidentIDs")
+action = demisto.args().get("action")
+if action not in ["link", "unlink"]:
+    action = "link"
+counter = int(demisto.args().get("retryLimit"))
+res = demisto.executeCommand("linkIncidents", {"linkedIncidentIDs": linkedIncidentIDs, "action": action})
+while isError(res[0]) and counter > 0 and ("DB Version" in res[0]["Contents"]):
+    time.sleep(2)
```

Comment:
How can you use `time` without importing it?
We avoid using sleep statements throughout the code, why is this needed?

Reply:
time is imported in common
@doug-couch @beauchompers could you advise why this is needed?

Reply:
@omercnet @guyfreund the button was based on the LinkIncidentsWithRetry automation, which is already in official content.

Reply:
Not sure I am following the reason to have the `sleep` statement. Care to elaborate @beauchompers?

Reply:
@ShahafBenYakir we probably don't need it, as I said I based it on an existing automation (LinkIncidentsWithRetry), figured someone ran into an issue with DB locks?

Reply:
`LinkIncidentsWithRetry` is quite old and deviates from our standards, it was also written in JS so no issues with docker containers like in python, let's remove this.